addons/sale_stock/report/sale_order_report_templates.xml

```diff
@@ -2,19 +2,19 @@
 <odoo>
     <template id="report_saleorder_document_inherit_sale_stock" inherit_id="sale.report_saleorder_document">
```

Comment:
all the example I see in the ticket are from the invoice report (used to print account.move records), but this part of the commit is modifying the sale order report (used to print sale.order records) report

I'm not sure this should be changed